wagtail/admin/static_src/wagtailadmin/scss/layouts/compare-revisions.scss

```diff
@@ -1,13 +1,16 @@
 @import 'wagtailadmin/scss/helpers';
 
-$color-addition-dark: #a6f3a6;
-$color-addition-light: #ebffeb;
-$color-deletion-dark: #f8cbcb;
-$color-deletion-light: #ffebeb;
+
+:root {
+    --color-addition-dark: #a6f3a6;
+    --color-addition-light: #ebffeb;
+    --color-deletion-dark: #f8cbcb;
+    --color-deletion-light: #ffebeb;
+}
```

Comment:
I don’t really see the point in those colors being theme-able? At least to me they shouldn’t be defined at the root level, we don’t want them reused elsewhere.

Reply:
I went about this with the goal of replacing every sass variable with css variables, and the postcss polyfill plugin requires that all css variables be defined inside `:root`. We could explore other plugins that don't have that restriction (although they have other footguns).

My thinking in converting these was that if someone were to make a dark theme or some other more complete and ambitious style override they might need to override these colors as well.

Reply:
Very good point. I had completely missed that PostCSS plugin only supports colors on `:root`, that’s annoying.